CPS-0012/README.md

```diff
@@ -13,24 +14,68 @@ Created: 2023-11-27
 
 ## Abstract
 
-Cardano lacks a standardized query layer. This leads to suboptimal tooling, dApp and wallet architecture.
+Blockchains lack the properties of classical databases, as storage is expensive and queries are complex.
```

Comment:
I still don't get what this sentence tries to convey. Which properties, for example?
What queries are complex and why?

Reply:
Indexers provide access via a relational db and/or a better API than the one node has.

Reply:
im just trying to _very briefly_ introduce why we have query layer providers

Reply:
@klntsky feel free to remove this, if youd prefer :)

Reply:
I have removed